.devcontainer/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/
 
 # [Optional] If your pip requirements rarely change, uncomment this section to add them to the image.
```

Comment:
This comment also seems suspect now as the section below is not currently commented. 

Should the comment also be removed?

Reply:
Y'a you can